src/Leafletjs/Resources/assets/src/controller.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import { Controller } from 'stimulus';
+
+export default class extends Controller {
+    static targets = ['placeholder'];
+
+    connect() {
+        import('leaflet').then((L) => {
+            let mapOptions = this._extractOptions('map-options');
+
+            this.map = L.map(this.placeholderTarget, mapOptions).setView(this._coordinates(), this.data.get('zoom'));
+
+            let layer = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
```

Comment:
I think it's good to also make the tile server configurable, to allow other map styles (or other OSM tile servers if e.g. their usage policy doesn't comply with your usage).